app/src/main/java/com/udacity/nanodegree/blooddonation/util/validation/PhoneNumberValidation.java

```diff
@@ -0,0 +1,39 @@
+package com.udacity.nanodegree.blooddonation.util.validation;
+
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.View;
+import android.widget.EditText;
+
+/**
+ * Created by tavishjain on 29-04-2018.
+ */
+
+public class PhoneNumberValidation {
```

Comment:
Why create a new class for PhoneNumberValidation. There is already a util class, just move the logic there. And user the same.

Also make use of Pattern.Phone regex.

Reply:
@ritesh-singh I dont know about Pattern.Phone regex

Reply:
https://stackoverflow.com/questions/22505336/email-and-phone-number-validation-in-android